skaha/src/main/java/org/opencadc/skaha/context/ResourceContexts.java

```diff
@@ -100,35 +102,45 @@ public class ResourceContexts {
 
     public ResourceContexts() {
         try {
-            PropertiesReader reader = new PropertiesReader("k8s-resources.properties");
-            MultiValuedProperties mvp = reader.getAllProperties();
-            defaultRequestCores = Integer.valueOf(mvp.getFirstPropertyValue("cores-default-request"));
-            defaultLimitCores = Integer.valueOf(mvp.getFirstPropertyValue("cores-default-limit"));
-            defaultCores = Integer.valueOf(mvp.getFirstPropertyValue("cores-default"));
-            defaultCoresHeadless = Integer.valueOf(mvp.getFirstPropertyValue("cores-default-headless"));
-            defaultRequestRAM = Integer.valueOf(mvp.getFirstPropertyValue("mem-gb-default-request"));
-            defaultLimitRAM = Integer.valueOf(mvp.getFirstPropertyValue("mem-gb-default-limit"));
-            defaultRAM = Integer.valueOf(mvp.getFirstPropertyValue("mem-gb-default"));
-            defaultRAMHeadless = Integer.valueOf(mvp.getFirstPropertyValue("mem-gb-default-headless"));
-            String cOptions = mvp.getFirstPropertyValue("cores-options");
-            String rOptions = mvp.getFirstPropertyValue("mem-gb-options");
-            String gOptions = mvp.getFirstPropertyValue("gpus-options");
-            
-            for (String c : cOptions.split(" ")) {
-                availableCores.add(Integer.valueOf(c));
-            }
-            for (String r : rOptions.split(" ")) {
-                availableRAM.add(Integer.valueOf(r));
-            }
-            for (String g : gOptions.split(" ")) {
-                availableGPUs.add(Integer.valueOf(g));
-            }
+            FileReader reader = new FileReader(getResourcesFile("k8s-resources.json"));
```

Comment:
Just to ensure that `FileReader` is properly closed, please encase it in the `try` right above it:
```java
try (final FileReader reader = new FileReader(getResourcesFile("k8s-resources.json"))) {
...
```